### PR TITLE
Fix unsubmittable 2FA Form on react apps

### PR DESCRIPTION
**Pull request by gregorylegarec**
It appears that we are not able to submit the 2FA Form in React apps.

Adding an onClick handler on submit button fix both submit by clik and with keyboard.

An issue will be made to investigate this behaviour further.

### CODE REVIEW

**Comment by CPatchane** [comment]
I noticed that when the form was not working, submitting with the Enter key was not working either, it that PR fix this issue?

File: packages/cozy-harvest-lib/src/components/TwoFAForm/index.jsx [hunk 1]
```diff
@@ -89,6 +90,7 @@ export class TwoFAForm extends PureComponent {
               busy={submitting}
               disabled={submitting || !twoFACode}
               extension="full"
+              onClick={this.handleSubmit}
```
Comment (Crash--) [inline packages/cozy-harvest-lib/src/components/TwoFAForm/index.jsx]: Did you tried to add a `type=submit`?

Comment (CPatchane) [reply packages/cozy-harvest-lib/src/components/TwoFAForm/index.jsx]: The CozyUI button is `type=submit` by default

Comment (ptbrowne) [reply packages/cozy-harvest-lib/src/components/TwoFAForm/index.jsx]: This is a bit surprising !

**Comment by gregorylegarec** [comment]
@CPatchane Yes, as explained in the PR description.

File: packages/cozy-harvest-lib/src/components/TwoFAForm/index.jsx (hunk 1 above)
Comment (drazik) [reply packages/cozy-harvest-lib/src/components/TwoFAForm/index.jsx]: Actually `submit` is the default type for raw `<button>`s. See https://codesandbox.io/s/frosty-leaf-dkfb9

**Pull request merged**